app.js

```diff
@@ -204,4 +204,44 @@ contactBtn.addEventListener('click', (e) => {
   } else {
     warning.innerHTML = '';
   }
-});
\ No newline at end of file
+});
+
+// Local Storage
+const formData = {
+  name: '',
+  email: '',
+  textArea: '',
+};
+
+function fillDatas() {
+  let storageData = localStorage.getItem('form-data');
+  storageData = JSON.parse(storageData);
+  document.getElementById('form-name').value = storageData.name;
+  document.getElementById('form-email').value = storageData.email;
+  document.getElementById('text-area').value = storageData.textArea;
+}
+
+function populateDatas() {
+  formData.name = document.getElementById('form-name').value;
+  formData.email = document.getElementById('form-email').value;
+  formData.textArea = document.getElementById('text-area').value;
+  localStorage.setItem('form-data', JSON.stringify(formData));
+  fillDatas();
+}
+
+const inputList = [];
+inputList.push(document.getElementById('form-name'));
+inputList.push(document.getElementById('form-email'));
+inputList.push(document.getElementById('text-area'));
+
+inputList.forEach((value) => {
+  value.onchange = function trig() {
+    populateDatas();
+  };
+});
+
+if (!localStorage.getItem('form-data')) {
+  populateDatas();
+} else {
+  fillDatas();
+}
```

Comment:
- [ ] [optional] It would be nice if you added to delete the data in local storage after the user submits the data